src/controllers/trainee/routes.ts

```diff
@@ -13,11 +13,11 @@ const traineeRouter: Router = Router();
 traineeRouter.route('/')
     .get(authMiddleWare('getUsers', 'read'), validationHandler(validation.get), traineeController.get)
     // .get(validationHandler(validation.get),traineeController.get)
-    .post(authMiddleWare('getUsers', 'write'), validationHandler(validation.create), traineeController.create)
+    .post(authMiddleWare('getUsers', 'read'), validationHandler(validation.create), traineeController.create)
```

Comment:
for post route use write in authMiddleWare

Reply:
done